### PR TITLE
Installation instructions linux

### PR DESCRIPTION
**Pull request by blaisb**
# Description of the problem

- The installation instructions on Linux were a bit dated. 

# Description of the solution
- I updated them to simplify them. I removed the whole deal.II manual installation thingy because it is rarely needed anymore. If you need it, you most likely already know how to do it. I feel the new instructions are really simpler.

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -28,9 +28,8 @@ Lethe is named after the river of forgetfulness, which, according to
    in order to forget their earthly life.
 
 Lethe is described `here <https://doi.org/10.1016/j.softx.2020.100579>`_.
-It originally began as a weekend project, but slowly grew into CFD
-software used in actual research.
-It is still an immature project, but is under active development.
+It originally began as a weekend project, but slowly grew into CFD, DEM and CFD-DEM
```
Comment (voferreira) [inline doc/source/index.rst:31]:
```suggestion
It originally began as a weekend project, but slowly grew into a robust and performative CFD, DEM and CFD-DEM
```

File: doc/source/index.rst [hunk 2]
```diff
@@ -28,9 +28,8 @@ Lethe is named after the river of forgetfulness, which, according to
...
+software used in actual research. Lethe is under active development.
```
Comment (voferreira) [inline doc/source/index.rst:32]:
```suggestion
software. Lethe is under active development.
```

File: doc/source/installation/regular_installation.rst [hunk 3]
```diff
@@ -6,30 +6,19 @@ Regular Installation on Linux
...
-* Lethe installation steps:
-  1. Installation of deal.II dependencies (MPI, numdiff, p4est, trilinos and METIS)
-  2. Installation of the deal.II library
-  3. Installation of Lethe
-
-* Methods to install deal.II and its dependencies:
-  1. Through Advanced Packaging Tool (apt): **this is by far the easiest way to proceed**, since it requires much less manual intervention.
+**Lethe installation steps:**
+  
+1. Installation of deal.II
```
Comment (voferreira) [inline doc/source/installation/regular_installation.rst:15]: This part could have an hyperlink to the steps

File: doc/source/installation/regular_installation.rst [hunk 4]
```diff
@@ -6,30 +6,19 @@ Regular Installation on Linux
...
 
-  2. through candi shell script (`candi github page <https://github.com/dealii/candi>`_). Even if you do not want to use candi to install deal.II, you can use it for the dependencies.
-
-  3. manually
-
-.. warning::
-  On a single core computer with 8GB of RAM, count up to 8 hours for a first installation, and 3 hours for a deal.II update. On a machine with 16 cores and 32GB of RAM, this process will take less than an hour or so. The installation of deal.II and its dependencies (especially trilinos), can be extremely RAM consuming. Installation on a machine with less than 8GB of RAM is difficult at best, impossible at worst.
+**Installing deal.II and its dependency:**
+  
+1. Through candi (recommended for developers)
```
Comment (voferreira) [inline doc/source/installation/regular_installation.rst:20]: Here I would add an hyperlink too

File: doc/source/installation/regular_installation.rst [hunk 5]
```diff
@@ -81,127 +105,19 @@ From the candi folder, the installation of candi can be launched using:
...
 
 
 After installation, you should have a ``deal.II-candi`` folder in the installation prefix directory, with the dealii folder of the desired version (see section :ref:`update-dealii`), as well as the required dependencies (p4est, trilinos, etc.).
 
-After installation, add an environment variable to your ``.bashrc`` either manually or through the following command:
+After installation, add the following lines variable to your ``.bashrc`` :
 
 .. code-block:: text
   :class: copy-button
-
```
Comment (voferreira) [inline doc/source/installation/regular_installation.rst:94]: Why remove the step-by-step installation procedure? I think I'd rather keep it.

**Review by voferreira — COMMENTED** [review]
It is good to update the installation procedure every once in a while. I think I would just keep the step-by-step deal.II installation guide. I will test it before approving, but here are the first comments.

**Comment by blaisb** [comment]
@voferreira I can keep the detailed deal.II instructions if you feel they are necessary. I think it kinda distracts from the whole thing (I would like to keep the page short), but if. you feel it's useful

**Comment by voferreira** [comment]
> @voferreira I can keep the detailed deal.II instructions if you feel they are necessary. I think it kinda distracts from the whole thing (I would like to keep the page short), but if. you feel it's useful

I agree. What do you think of placing it at the end of the instructions?

**Review by voferreira — APPROVED** [review]
Good to go. Please consider my last comment :)

**Comment by blaisb** [comment]
I'll add the hyper links and add the additional removed block at the end as a supplementary information kinda

File: doc/source/installation/regular_installation.rst [hunk 6]
```diff
@@ -64,12 +57,49 @@ This should output several information about the installed version. Everything w
...
+    PACKAGES="${PACKAGES} once:numdiff"
+    PACKAGES="${PACKAGES} once:opencascade"
+    PACKAGES="${PACKAGES} once:parmetis"
+    PACKAGES="${PACKAGES} once:p4est"
+    PACKAGES="${PACKAGES} once:trilinos"
+    PACKAGES="${PACKAGES} dealii"
+
+Other packages can be disabled by simply commenting out the lines (adding an ``#`` at the beggining of the lines)
+
+To ensure that the the lethe test suite works, deal.II must be configured with p4est version 2.2.1. In the subfolder ``deal.II-toolchain/packages/``, open the ``p4est.package`` file with a text editor and change the following lines:
```
Comment (oguevremont) [inline doc/source/installation/regular_installation.rst:84]:
```suggestion
To ensure that the the Lethe test suite works, deal.II must be configured with p4est version 2.2.1. In the subfolder ``deal.II-toolchain/packages/``, open the ``p4est.package`` file with a text editor and change the following lines:
```

File: doc/source/installation/regular_installation.rst [hunk 7]
```diff
@@ -81,82 +111,82 @@ From the candi folder, the installation of candi can be launched using:
...
---------------------------------------
-.. note:: 
-  If you have installed deal.II through candi, you can skip right away to :ref:`install-lethe`
+.. _install-lethe:
 
-We first need to install the deal.II dependencies.
+Installing Lethe 
+-------------------------------
 
+Clone lethe from the `Lethe official repository <https://github.com/lethe-cfd/lethe>`_.
```
Comment (oguevremont) [inline doc/source/installation/regular_installation.rst:134]:
```suggestion
Clone Lethe from the `Lethe official repository <https://github.com/lethe-cfd/lethe>`_.
```

**Review by oguevremont — APPROVED** [review]
It's a nice simplification of this page. I only have typos corrections, but the content is good

File: examples/incompressible-flow/2d-lid-driven-cavity/cavity.prm [hunk 8]
```diff
@@ -29,7 +29,7 @@ subsection mesh
   set type               = dealii
   set grid type          = hyper_cube
   set grid arguments     = 0 : 1 : true
-  set initial refinement = 6
+  set initial refinement = 8
```
Comment (blaisb) [inline examples/incompressible-flow/2d-lid-driven-cavity/cavity.prm:32]:
```suggestion
  set initial refinement = 6
```

**Pull request merged**